src/index.ts

```diff
@@ -27,8 +20,8 @@ const walletRegexes: Partial<Record<WalletType, RegExp>> = {
     [WalletType.BITCOIN]: testBitcoin(),
 };
 
-// Validate wallet address function
-const validateWalletAddress = (address: string): WalletType | null => {
+// Function to get wallet address type
+const getWalletAddressType = (address: string): WalletType | null => {
     // Check if the address is a Bitcoin address first
     if (walletRegexes[WalletType.BITCOIN]?.test(address)) {
```

Comment:
Can you separate the validation functions and regexes into 1 file? So all EVM, bitcoin and solana regex and validation process would go into 1 file and the index file would just be clean and simple

Reply:
Updated code with proper folder structure